maven-plugin-api/pom.xml

```diff
@@ -34,6 +34,10 @@ under the License.
   <description>The API for plugins - Mojos - development.</description>
 
   <dependencies>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
```

Comment:
This one may be optional? Or provided? 
As it is needed during build and runtime (will be provided) but plugin implementor may not include it.

Reply:
+1 for provided, best would also be to drop it from the published pom since we likely don't want plugin writer to use slf4j for their mojo logger but getLog() API

Reply:
wdym by getLog() API?

Reply:
Mojo#getLog is deprecated

Reply:
No, please see the mailing list discussion about it and why slf4j must not leak in mojo/plugin-api.

Reply:
I agree then logging system should not leak in this point

Reply:
Please take a peek at 2645d0995dcdbffbbea36a44c9a32a04d2d99183